EIPS/eip-7620.md

```diff
@@ -243,18 +250,40 @@ function txcreate(a, b, c, d, e) -> f {
     
 ## Rationale
 
+### Data section appending
+
 The data section is appended to during contract creation and also its size needs to be updated in the header. Alternative designs were considered, where:
 
 - additional section kinds for the data were introduced
 - additional fields describing a subcontainer were introduced
 - data section would be written over as opposed to being appended to, requiring it to be filled with 0 bytes prior to deployment
     
 All of these alternatives either complicated the otherwise simple data structures or took away useful features (like the dynamically sized portion of the data section).
+
+### `TXCREATE`
+
+An alternative design was discussed, where instead of `TXCREATE`, `InitcodeTransaction` and the Deployer Contract, the legacy-like creation transactions (with empty `to`) continue to be available to send EOF initcontainers in transactions and perform deployment of EOF contracts.
```

Comment:
```suggestion
An alternative design was discussed, where instead of `TXCREATE`, `InitcodeTransaction` and the Creator Contract, the legacy-like creation transactions (with empty `to`) continue to be available to send EOF initcontainers in transactions and perform deployment of EOF contracts.
```